src/node/identity.h

```diff
@@ -10,9 +10,20 @@
 
 namespace ccf
 {
-  struct NetworkIdentity
+  struct Identity
   {
+    enum class IdentityType
+    {
+      NORMAL,
```

Comment:
I don't think NORMAL is very descriptive any chance this can be renamed?

Reply:
I agree, but I can't find a good name. What would you call the normal/current concept of identity?

Reply:
Replicated/Sharded|Distributed?

Reply:
FWIW, I think "Byzantine" is also very confusing, because it implies malicious rather than "Byzantine Fault Tolerant". Describing whether the identity is copied/replicated in each node, or sharded/split seems more accurate.